src/css/variables.less

```diff
@@ -117,3 +129,14 @@
  */
 @popover-border-radius: 2px;
 @popover-width: @base-spacing-unit * 25;
+
+/* ==============
+ * Modal
+ * ==============
+ */
+@modal-border-radius: 6px;
+@modal-body-padding: 20px;
+@modal-footer-padding: 33px;
+@modal-dialog-padding: @modal-footer-padding;
+@modal-dialog-width: @base-spacing-unit * 67;
+@modal-dialog-height: @base-spacing-unit * 29;
```

Comment:
Also, I'm not too sure about these. 29 and 67 feel very arbitrary. I'd rather have a dedicated variable for these measures, if they have to match a certain value.